libs/datasets/sources/state_provided_testing_dataset.py

```diff
@@ -0,0 +1,40 @@
+import dataclasses
+from libs.datasets import data_source
+from datapublic.common_fields import CommonFields
+from libs.datasets.sources import can_scraper_helpers as ccd_helpers
+from libs.datasets.timeseries import MultiRegionDataset
+
+
+class StateProvidedTestingDataset(data_source.CanScraperBase):
+    """Data source connecting to state health department sourced testing data."""
+
+    SOURCE_TYPE = "StateProvidedTesting"
+
+    VARIABLES = [
+        ccd_helpers.ScraperVariable(
+            variable_name="pcr_tests_positive",
+            measurement="cumulative",
+            provider="state",
+            unit="specimens",
+            common_field=CommonFields.POSITIVE_TESTS,
```

Comment:
I think we could / should be using POSITIVE_TESTS_VIRAL and TOTAL_TESTS_VIRAL since these are labeled as PCR tests.  Any reason you didn't?

Changing it might cause a conflict with the existing `DivisionMethod` in libs/metrics/test_positivity.py though?  So if this is already tested and you want to ship it as-is, I'm fine with that.

Reply:
Ah, no reason I just didn't fully understand the defns I think--will update thanks!

Reply:
As for the conflict, I think it wouldn't have been an issue (I ended up creating `NEW_POSITIVE/TOTAL_TESTS_VIRAL` to signify we're using new, not cumulative data) because (if I'm understanding correctly) the existing division method is looking for those variables specifically in the 'positiveTestsViral_totalTestsViral` dataset (which doesn't seem to exist anymore, so maybe we should nuke it?) whereas the new method would look in the newly created dataset

Reply:
Oh, I see. Heh, yeah. I wasn't sure what that `positiveTestsViral_totalTestsViral` was referring to. 👍